src/Uno.UI/Microsoft/UI/Xaml/Controls/ColorPicker/ColorPickerSliderAutomationPeer.cs

```diff
@@ -0,0 +1,136 @@
+		{
+			get => false;
+		}
+
+		public string Value
+		{
+			get
+			{
+				ColorPickerSlider owner = this._owner;
+				DependencyObject currentObject = owner;
+				ColorPicker owningColorPicker = null;
+
+				// Uno Doc: Re-written in C# to avoid assignment during while expression evaluation
+				while (currentObject != null)
+				{
+					owningColorPicker = currentObject as ColorPicker;
+
+					if (owningColorPicker != null) { break; }
+
+					currentObject = VisualTreeHelper.GetParent(currentObject);
+				}
+
+				if (owningColorPicker != null)
+				{
+					Color color = owningColorPicker.Color;
+					double sliderValue = owner.Value;
+
+					return GetValueString(color, (Int32)(Math.Round(sliderValue)));
+				}
+				else
+				{
+					return string.Empty;
+				}
+			}
+		}
+
+		public void SetValue(string value)
+		{
+			// Uno Doc: Not used so commented out
+			// MUX_ASSERT(false); // Not implemented.
+		}
+
+		public void RaisePropertyChangedEvent(Color oldColor, Color newColor, int oldValue, int newValue)
+		{
+			string oldValueString = GetValueString(oldColor, oldValue);
+			string newValueString = this.GetValueString(newColor, newValue);
+
+			base.RaisePropertyChangedEvent(ValuePatternIdentifiers.ValueProperty, oldValueString, newValueString);
+		}
+
+		private string GetValueString(Color color, int value)
+		{
+			if (DownlevelHelper.ToDisplayNameExists())
+			{
+				string resourceStringWithName;
+				switch (this._owner.ColorChannel)
+				{
+					case ColorPickerHsvChannel.Hue:
+						resourceStringWithName = ResourceAccessor.GetLocalizedStringResource(ResourceAccessor.SR_ValueStringHueSliderWithColorName);
+						 break;
+					case ColorPickerHsvChannel.Saturation:
+						resourceStringWithName = ResourceAccessor.GetLocalizedStringResource(ResourceAccessor.SR_ValueStringSaturationSliderWithColorName);
+						break;
+					case ColorPickerHsvChannel.Value:
+						resourceStringWithName = ResourceAccessor.GetLocalizedStringResource(ResourceAccessor.SR_ValueStringValueSliderWithColorName);
+						break;
+					default:
+						return string.Empty;
+				}
+
+				return string.Format(
+					resourceStringWithName,
+					value,
+					ColorHelper.ToDisplayName(color));
+			}
+			else
+			{
+				string resourceStringWithoutName;
+				switch (this._owner.ColorChannel)
+				{
+					case ColorPickerHsvChannel.Hue:
+						resourceStringWithoutName = ResourceAccessor.GetLocalizedStringResource(ResourceAccessor.SR_ValueStringHueSliderWithoutColorName);
+						break;
+					case ColorPickerHsvChannel.Saturation:
+						resourceStringWithoutName = ResourceAccessor.GetLocalizedStringResource(ResourceAccessor.SR_ValueStringSaturationSliderWithoutColorName);
+						break;
+					case ColorPickerHsvChannel.Value:
+						resourceStringWithoutName = ResourceAccessor.GetLocalizedStringResource(ResourceAccessor.SR_ValueStringValueSliderWithoutColorName);
+						break;
+					default:
+						return string.Empty;
+				}
+
+				return string.Format(
```

Comment:
Codacy found an issue: [Use the overload that takes a 'CultureInfo' or 'IFormatProvider' parameter.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6121259)